service/src/test/java/uk/gov/hmcts/reform/fpl/utils/NotifyAdminOrderIssuedTestHelper.java

```diff
@@ -0,0 +1,119 @@
+package uk.gov.hmcts.reform.fpl.utils;
+
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.MapDifference;
+import com.google.common.collect.Maps;
+import org.apache.commons.codec.binary.Base64;
+import org.json.JSONObject;
+import org.mockito.ArgumentCaptor;
+import uk.gov.hmcts.reform.fpl.enums.IssuedOrderType;
+import uk.gov.hmcts.reform.fpl.model.Address;
+import uk.gov.hmcts.reform.fpl.model.Representative;
+import uk.gov.hmcts.reform.fpl.model.common.Element;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static java.nio.charset.StandardCharsets.ISO_8859_1;
+import static org.assertj.core.api.Assertions.assertThat;
+import static uk.gov.hmcts.reform.fpl.enums.IssuedOrderType.CMO;
+import static uk.gov.hmcts.reform.fpl.enums.RepresentativeServingPreferences.POST;
+import static uk.gov.hmcts.reform.fpl.utils.ElementUtils.wrapElements;
+import static uk.gov.hmcts.reform.fpl.utils.EmailNotificationHelper.formatCaseUrl;
+
+public class NotifyAdminOrderIssuedTestHelper {
+
+    private NotifyAdminOrderIssuedTestHelper() {
+    }
+
+    private static final byte[] PDF = {1, 2, 3, 4, 5};
+    private static final String LOCAL_AUTHORITY_NAME = "Example Local Authority";
+
+    public static MapDifference<String, Object> verifyNotificationSentToAdminWhenOrderIssued(
+        ArgumentCaptor<Map<String, Object>> dataCaptor, IssuedOrderType issuedOrderType) {
+
+        Map<String, Object> resultData = new HashMap<>(dataCaptor.getValue());
+        Map<String, Object> expectedParameters = issuedOrderType == CMO
+            ? getExpectedCMOParametersForAdminWhenRepresentativesNeedServingByPost() :
+            getExpectedParametersForAdminWhenRepresentativesNeedServingByPost();
+
+        assertThat(((JSONObject) resultData.get("caseUrlOrDocumentLink")).get("file")).isEqualTo(
+            ((JSONObject) expectedParameters.get("caseUrlOrDocumentLink")).get("file"));
+
+        resultData.remove("caseUrlOrDocumentLink");
+        expectedParameters.remove("caseUrlOrDocumentLink");
+
+        return Maps.difference(expectedParameters, resultData);
+    }
+
+    public static Map<String, Object> getExpectedParametersForAdminWhenNoRepresentativesServedByPost() {
+        return ImmutableMap.<String, Object>builder()
+            .put("cmoCallout", "")
+            .putAll(commonParametersNoPostingNeeded())
+            .build();
+    }
```

Comment:
Map.of()

Reply:
why is this preferred? do you mean like this:
```
        Map<String, Object> expectedMap = new HashMap<>();
        expectedMap.put("cmoCallout", "");
        expectedMap.putAll(commonParametersNoPostingNeeded());
        return expectedMap;
```